third_party/libunwindstack/Unwinder.cpp

```diff
@@ -121,8 +121,36 @@ static bool ShouldStop(const std::vector<std::string>* map_suffixes_to_ignore,
                    map_name.substr(pos + 1)) != map_suffixes_to_ignore->end();
 }
 
-void Unwinder::Unwind(const std::vector<std::string>* initial_map_names_to_skip,
-                      const std::vector<std::string>* map_suffixes_to_ignore) {
+static bool ShouldStop(
+    const std::map<std::string, std::map<uint64_t /*function_start*/, uint64_t /*size*/>>*
+        functions_to_stop,
+    const std::string& map_name, uint64_t rel_pc) {
+  if (functions_to_stop == nullptr) {
+    return false;
+  }
+  auto functions_it = functions_to_stop->find(map_name);
```

Comment:
Using the map name here immediately brings to mind that the case of anonymous maps containing sections of PEs with "bad" alignment won't work. Maybe it's fine but since we send this information from the client, can't we just use the absolute address (and the size) instead? It would also save use from having to keep module names around altogether and hence simplify the parameter.

Reply:
An alternative would be to use `MapInfo::GetObject` as that also works for anonymous maps, but we currently don't save the file name in `Elf` or `PeCoff`.